tema21ian/src/ExerciseThree/Time.java

```diff
@@ -0,0 +1,26 @@
+package ExerciseThree;
+
+import java.util.Scanner;
+
+public class Time {
+    public static void main(String[] args) {
+        System.out.println("What time is it?");
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Number for hour is:");
+        int hour = sc.nextInt();
+        System.out.println("Number for minute is:");
+        int min = sc.nextInt();
+        displayTime(hour,min);
+
+    }
+
+    public static void displayTime(int hour, int min){
```

Comment:
in order to test this method you would need to check what the method outputs (to System.out).

If you would change this method to "public static String timeToString(int hour, int min)" you could test this method and see if it returns the proper String. 
Then the System.out would be done by the main method.